factor/grace/lang/en/factor_grace.php

```diff
@@ -29,6 +29,8 @@
 $string['settings:forcesetup_help'] = 'Forces a user to the preferences page to setup MFA when the gracemode period expires. If set to off, users will be unable to authenticate when the grace period expires.';
 $string['settings:graceperiod'] = 'Grace period';
 $string['settings:graceperiod_help'] = 'Period of time when users can access Moodle without configured and enabled factors';
+$string['settings:ignorelist'] = 'Ignored factors';
+$string['settings:ignorelist_help'] = 'Any factors here will not be counted by grace when deciding whether to give points.';
```

Comment:
Sorry, but I can't understand what it is really means for the users :( Maybe add an example?